### DISCUSSION TITLE
🇩🇪 Sunny Shine - on the Funny Side of Life

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/eltikia.github.io#8] [Q&A] [answered]
Was bedeute die Videobeschreibungen?

**Comment by Stagyrite** [comment, marked as answer]
> Sunny ist allergisch gegen Alkohol, aber Cigaretten schmeckt ihm gut.

Sunny trägt immer L&M Zigaretten mit ihm. Erst er kauft eine Packung ein. Er will kein Glas Sekt. Wenn er trinkt Alkohol, seine Allergie äußert sich. Er werdet ohnmächtig und bekomme überall rote Pusteln.

> Dieses Computerspiel kann nicht gut enden, denn er wirfte ein Poster weg.

Sunny soll immer mit ihm ein Poster tragen. Er wirt das weg, und danach er muss bezahlen, aber er hat zu wenig Geld.

> Er hat auch weniger Geld als die Konkurenz.

Die Konkurrenz kauf ein Auto ein und das Ziel ist nicht erreicht.
